lib/new_relic/agent/instrumentation/sidekiq/extensions/delayed_class.rb

```diff
@@ -8,7 +8,7 @@
 #
 #       see https://github.com/mperham/sidekiq/issues/5076 for the discussion
 #       of the removal, which includes mentions of alternatives
-if Sidekiq::VERSION < '7.0.0'
+if defined?(Sidekiq::VERSION) && Sidekiq::VERSION < '7.0.0'
```

Comment:
This class is loaded by a simple require performed by `test/new_relic/agent/instrumentation/instrumentation_test.rb`, which operates without Sidekiq. As a result, we need to guard with an extra `defined?` check here.